front-end/src/utilities/EventUtilities.jsx

```diff
@@ -72,30 +72,39 @@ export const setUserAttending = async (eventID, usersAttending) => {
     }
 };
 
+//wrapped in a try catch and additional console.logs for better error handling
 export const updateEventDetails = async (eventID, title, eventStart, eventEnd, timeZone, eventType, eventVenue, eventVenueAddress, description, category, eventPhoto, virtualEventLink, location, eventCoordinates) => {
-    let response = await api.put(`events/${eventID}/`, {
-        "title" : title,
-        "event_start" : eventStart,
-        "event_end" : eventEnd,
-        "time_zone" : timeZone,
-        "event_type" : eventType,
-        "event_venue" : eventVenue,
-        "event_venue_address" : eventVenueAddress,
-        "description" : description,
-        "category" : category,
-        "event_photo" : eventPhoto,
-		"virtual_event_link": virtualEventLink,
-        "location": location,
-        "coordinates": eventCoordinates
-    });
-    console.log(response.status);
-    if (response.status === 200) {
-        return true;
-    } else {
-        console.log("error:", response.data);
+    try {
+        let response = await api.put(`events/${eventID}/`, {
+            "title": title,
+            "event_start": eventStart,
+            "event_end": eventEnd,
+            "time_zone": timeZone,
+            "event_type": eventType,
+            "event_venue": eventVenue,
+            "event_venue_address": eventVenueAddress,
+            "description": description,
+            "category": category,
+            "event_photo": eventPhoto,
+            "virtual_event_link": virtualEventLink || null,  //to satisfy backend requirements 
+            "location": location,
+            "eventCoordinates": eventCoordinates || "" //to satisfy backend requirements 
```

Comment:
If you're still working this - this should be "coordinates": eventCoordinates || []

Reply:
nice catch! uploading now